push_live.sh

```
@@ -0,0 +1,89 @@
+	# Strip the source root to get the relative-from-/ portion.
+	rel="${abs#"$SOURCE_ROOT"/}"
+	if [ "$rel" = "$abs" ]; then
+		echo "Not inside $SOURCE_ROOT: $arg" >&2
+		exit 1
+	fi
+	remote="/$rel"
+	if [ -d "$abs" ]; then
+		echo "==> $arg/  ->  $UNRAID_HOST:$remote/"
+		# Trailing /. on src + trailing / on dst copies *contents* of the
+		# directory into the existing remote directory.
+		ssh "$UNRAID_HOST" "mkdir -p \"$remote\""
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)